### ISSUE TITLE
bintray propertie error when importing

### ISSUE THREAD

**Issue by Testator** [Codigami/CFAlertDialog#30]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/Codigami/CFAlertDialog/blob/develop/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/Codigami/CFAlertDialog/blob/develop/README.md)
* [ ] I have searched for a similar issue in the [project](https://github.com/Codigami/CFAlertDialog/issues) and found none

### Issue Info

 Info                          | Value                               |
-------------------------------|-------------------------------------|
 Platform Name                 | e.g. Android
 Platform Version              | e.g. 6.0
 CFAlertDialog Version         | e.g. 2.1.1
 Integration Method            | e.g. gradle / manually
 Xcode Version                 | e.g. Android Studio 2.3
 Repro rate                    | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj       | e.g. does it happen with our demo project?
 Demo project link             | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

error when trying to compile
properties.load(project.rootProject.file('bintray.properties').newDataInputStream()) 
does not exist

changing back to 
properties.load(project.rootProject.file('local.properties').newDataInputStream())
compiles

any clue why this is happenning?

**Comment by rahul-s** [comment]
Can you explain to me the need for you to build the library code..?
Instead you can directly use the library in your project with this command
`implementation 'com.crowdfire.cfalertdialog:cfalertdialog:1.3.1'`  in your module's `build.gradle`.

**Comment by Testator** [comment]
When you import the demo into android studio it gives you an error message
when compiling
